### ISSUE TITLE
False negative for `CKV2_AZURE_8`: "Ensure the storage container storing the activity logs is not publicly accessible"

### ISSUE THREAD

**Issue by Saibamen** [bridgecrewio/checkov#4873]
**Describe the issue**
False negative for `CKV2_AZURE_8: "Ensure the storage container storing the activity logs is not publicly accessible"`

The default value for `container_access_type` is `private`, so for scripts without this key, this check is false negative.

Checkov guide:
https://docs.bridgecrew.io/docs/ensure-the-storage-container-storing-the-activity-logs-is-not-publicly-accessible

Terraform documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_container#argument-reference

**Version (please complete the following information):**
 - Checkov Version: 2.3.158

**Comment by gruebel** [comment]
hey @Saibamen thanks for reaching out.

I tested it with this setup and I the check passes

```hcl

resource "azurerm_storage_container" "default" {
  name                  = "vhds"
  storage_account_name  = azurerm_storage_account.example.name
}

resource "azurerm_storage_account" "example" {
  name                     = "examplesa"
  resource_group_name      = azurerm_resource_group.main.name
  location                 = azurerm_resource_group.main.location
  account_tier             = "Standard"
  account_replication_type = "GRS"
}

resource "azurerm_monitor_activity_log_alert" "example" {
  name                = "example-activitylogalert"
  resource_group_name = azurerm_resource_group.main.name
  scopes              = [azurerm_resource_group.main.id]
  description         = "This alert will monitor a specific storage account updates."

  criteria {
    resource_id    = azurerm_storage_account.example.id
    operation_name = "Microsoft.Storage/storageAccounts/write"
    category       = "Recommendation"
  }

  action {
    action_group_id = azurerm_monitor_action_group.main.id

    webhook_properties = {
      from = "terraform"
    }
  }
}
```

**Comment by Saibamen** [comment]
Sorry, my `azurerm_storage_container` resource was set as `container_access_type = "blob"`

**Comment by gruebel** [comment]
ok, great, then I close the issue 🙂
